### ISSUE TITLE
documentation on Tiny Yolo v3

### ISSUE THREAD

**Issue by chrisrapson** [pjreddie/darknet#1747]
Is there any documentation on how the layers are connected for Tiny Yolo v3? The articles only seem to describe the main trunk, i.e. convolutional/residual/pooling/connected/softmax layers, so I'm not entirely sure which layers the `yolo` layers connect to, and whether there are any skip connections. Likewise, `yolo` layers are missing from the description on https://pjreddie.com/darknet/tiny-darknet/, but they are definitely in the [cfg file](https://github.com/pjreddie/darknet/blob/master/cfg/yolov3-tiny.cfg) (and the .c file).

I'm asking because I wanted to try out a few of my own modifications, and my first instinct was to import my existing model into tensorflow. There are a couple of projects out there that convert yolo to tf, and the first two I tried gave different graphs (starting from the same cfg file). One has a skip connection branching off after a conv layer and one branches after a maxpool layer. Only one of them can be right, but I can't find any documentation that tells me which one. In the first one, the connections to the `yolo` layers also look fairly complex, and those layers are missing completely from the second one. I'm about to do a deep dive into the code, but I thought I'd ask here in case someone can point me towards an easy answer.

![tensorboard_graph](https://user-images.githubusercontent.com/4982550/62091863-1a3dbb00-b2c7-11e9-96a8-8ea09dc576a8.png)
![tensorboard_graph](https://user-images.githubusercontent.com/4982550/62091867-1dd14200-b2c7-11e9-8092-3bc6a0154967.png)

P.S. the readme still says to put questions and issues on the google groups page, but that page is showing me a "Banned Content Warning"

**Comment by chrisrapson** [comment]
Never mind, I think I found the answer here:
[AlexeyAB:#487](https://github.com/AlexeyAB/darknet/issues/487#issuecomment-374902735)
